.travis.yml

```diff
@@ -0,0 +1,23 @@
+# according to the internet Python envs are hard and it's better to build them from scratch
+language: cpp
+os:
+- osx
+sudo: required
+install:
+- if [ "$TRAVIS_OS_NAME" = "osx" ]; then bash package/prepare_osx.sh; fi
+script:
+- python3 package/dependency_test.py
```

Comment:
This is a very basic test if main application loads. In the future it would be cool to have some real tests in here. This is run on every push, not on tag push.

Reply:
What does it mean on every push? Every time any one pushes anything to the repo? And then where do you see the output? (Sorry if it's a stupid question, I'm not sure I understood exactly how travis works yet :p)

Reply:
>What does it mean on every push? Every time any one pushes anything to the repo? 

Yes. Also, on every pull request made etc.

>And then where do you see the output?

Not a stupid question at all ;) You can see history of builds for my fork here, for example: https://travis-ci.org/katafrakt/manuskript/builds

Also, you'd need to enable Travis on main repository. It should be possible here: https://travis-ci.org/profile/olivierkes?offset=0